modules/feature-management/src/Volo.Abp.FeatureManagement.Domain/Volo/Abp/FeatureManagement/AbpFeatureManagementDomainModule.cs

```diff
@@ -24,7 +22,6 @@ public override void ConfigureServices(ServiceConfigurationContext context)
 
                 //TODO: Should be moved to the Tenant Management module
                 options.Providers.Add<TenantFeatureManagementProvider>();
-                options.ProviderPolicies[TenantFeatureValueProvider.ProviderName] = "AbpTenantManagement.Tenants.ManageFeatures";
```

Comment:
This should be restored based on my previous comment (for the FeatureAppService.cs)